src/SFA.DAS.Admin.Aan.Web/Controllers/ManageMembers/MemberProfileController.cs

```diff
@@ -93,6 +95,11 @@ private async Task<AmbassadorProfileViewModel> GetViewModel(Guid id, Guid adminU
         ambassadorProfileViewModel.RemoveMember.FirstName = memberProfiles.FirstName;
         ambassadorProfileViewModel.RemoveMember.RouteLink = Url.RouteUrl(RouteNames.RemoveMember, new { id })!;
 
+        if (ambassadorProfileViewModel.MemberInformation.MaturityStatus == MemberMaturityStatus.New && ambassadorProfileViewModel.MemberInformation.UserRole == Role.Apprentice && memberProfiles.Profiles.Any(x => x.ProfileId == ProfileIds.ReasonToJoinAmbassadorNetwork))
```

Comment:
Check for role should be first condition

Reply:
Done